packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -113,11 +115,16 @@ export function buildWebpackConfig(
     ? additionalOptions.logger.createChild('webpackConfigOptions')
     : new logging.NullLogger();
 
+
+  // At the moment, only the browser builder supports differential loading
+  // However this config generation is used by multiple builders such as dev-server
+  // const builderInfo = additionalOptions.builderInfo;
   const scriptTarget = tsConfig.options.target;
-  // todo enabe when differential loading is complete
-  // const differentialLoading = isDifferentialLoadingNeeded(projectRoot, scriptTarget);
+  // const differentialLoading =
```

Comment:
Not sure, if there is a better way apart from providing an addition parameter to the upper function and pass it down. 

But I think for the time being, this is enough